workflows/stairway.py

```diff
@@ -36,15 +36,17 @@ class StairwayPlotRunner(object):
     """
     Run stairway plots.
     """
+
     def __init__(self, workdir, stairway_dir, java_exe="java"):
         self.workdir = pathlib.Path(workdir)
         shutil.rmtree(self.workdir, ignore_errors=True)
         self.workdir.mkdir(parents=True)
         stairway_path = pathlib.Path(stairway_dir)
-        self.classpath = "{}:{}".format(stairway_path, stairway_path / "swarmops.jar")
+        self.classpath = "{}:{}".format(
+            stairway_path, stairway_path / "swarmops.jar")
```

Comment:
need to add `openjdk` into `environment.yml`

Reply:
added